<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>861</width>
      <height>741</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>801</width>
       <height>691</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/wallpaperMountain.jpg);
border-image: url(:/images/wallpaperMountainRegister.jpg);
border-radius: 40px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>781</width>
       <height>651</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,100);
border-radius: 30px;</string>
     </property>
     <property name="text">
      <string comment="Password"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(90, 233, 235, 1);</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nombreUsuarioRegister">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 4px solid rgba(105,118,132,255);
color: rgba(255,255,255,255);
padding-bottom: 7px;</string>
     </property>
     <property name="placeholderText">
      <string>Nombre del Usuario</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>550</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#registerButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color: rgba(255,255,255,210);
	border-radius: 10px;
}

QPushButton#registerButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
}

QPushButton#registerButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color:rgba(105,118,132,200);
}</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordRegister">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>460</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 4px solid rgba(105,118,132,255);
color: rgba(255,255,255,255);
padding-bottom: 7px;</string>
     </property>
     <property name="placeholderText">
      <string>Contrase√±a</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="correoRegister">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>390</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 4px solid rgba(105,118,132,255);
color: rgba(255,255,255,255);
padding-bottom: 7px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>Correo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dniRegister">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 4px solid rgba(105,118,132,255);
color: rgba(255,255,255,255);
padding-bottom: 7px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>DNI</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="apellidoUsuarioRegister">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 4px solid rgba(105,118,132,255);
color: rgba(255,255,255,255);
padding-bottom: 7px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Apellidos del Usuario</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
